src/components/IstioWizards/Slider/BootstrapSlider.tsx

```diff
@@ -74,12 +82,14 @@ class BootstrapSlider extends React.Component<Props> {
         this.props.onSlide(value);
         this.slider.setValue(value);
       };
+      const onSlideStop = value => {
+        value = value >= this.props.max ? this.props.max : value;
+        this.props.onSlideStop(value);
+        this.slider.setValue(value);
+      };
       this.slider.on('slide', onSlide);
-      this.slider.on('slideStop', onSlide);
+      this.slider.on('slideStop', onSlideStop);
       this.slider.setAttribute('formatter', this.props.formatter);
-      if (this.props && this.props.orientation && orientation) {
-        console.log('TODELETE orientation check');
```

Comment:
:D

Reply:
No points for these until a DNM label is removed :)